packages/apps/transfer/next.config.js

```diff
@@ -4,6 +4,15 @@ const nextTranslate = require('next-translate-plugin');
 module.exports = nextTranslate({
   reactStrictMode: true,
   pageExtensions: process.env.NODE_ENV === "production" ? ['(?<!(spec|test).)tsx'] : ['tsx'],
+  env: {
+    KADENA_API_TTIL: JSON.stringify(process.env.KADENA_API_TTIL),
+    KADENA_MAINNET_API: JSON.stringify(process.env.KADENA_MAINNET_API),
+    KADENA_MAINNET_NETWORKS: JSON.stringify(process.env.KADENA_MAINNET_NETWORKS),
+    KADENA_TESTNET_API: JSON.stringify(process.env.KADENA_TESTNET_API),
+    KADENA_TESTNET_NETWORKS: JSON.stringify(process.env.KADENA_TESTNET_NETWORKS),
+    GAS_PRICE: JSON.stringify(process.env.GAS_PRICE),
+    GAS_LIMIT: JSON.stringify(process.env.GAS_LIMIT),
+  },
```

Comment:
What's te reason to `JSON.stringify` everything? If I recall correctly, everything from `process.env` is already a `string`

Reply:
it's not required, it's just to ensure that the value received is a string on the different environments